src/admin-views/meta-box.php

```diff
@@ -9,238 +9,235 @@
 if ( ! defined( 'ABSPATH' ) ) {
 	die( '-1' );
 }
-
-$header_id  = get_post_meta( get_the_ID(), $this->image_header_field, true );
-$header_id  = ! empty( $header_id ) ? $header_id : '';
+$post_id = get_the_ID();
+$header_id = get_post_meta( $post_id, $this->image_header_field, true );
+$header_id = ! empty( $header_id ) ? $header_id : '';
 $header_img = '';
 if ( ! empty( $header_id ) ) {
 	$header_img = wp_get_attachment_image( $header_id, 'full' );
 }
 
 $modules = Tribe__Tickets__Tickets::modules();
+$total_tickets = Tribe__Tickets__Tickets_Handler::instance()->get_event_tickets_count( $post_id );
+
 ?>
 
-<table id="event_tickets" class="eventtable">
+<div id="event_tickets" class="eventtable">
 	<?php
 	wp_nonce_field( 'tribe-tickets-meta-box', 'tribe-tickets-post-settings' );
 
-	if ( get_post_meta( get_the_ID(), '_EventOrigin', true ) === 'community-events' ) {
+	// the main panel ?>
+	<div id="panel_base" class="ticket_panel panel_base">
+		<div class="tribe_sectionheader ticket_list_container">
+			<div class="ticket_table_intro">
+				<span class="ticket_form_total_capacity">
+					Total Event Capacity:
+					<span id="ticket_form_total_capacity_value">
+						<?php if ( 0 < $total_tickets ) {
+							echo absint( $total_tickets );
+						} else {
+							esc_html_e( 'unlimited' );
+						}?>
```

Comment:
Can you reformat this `if` statement so that the open/close PHP tags are on separate lines?

```php
<?php
if ( 0 < $total_tickets ) {
  echo absint( $total_tickets );
} else {
  esc_html_e( 'unlimited' );
}
<?php
```